tests/containers/award/AwardContainer-test.jsx

```diff
@@ -365,105 +367,231 @@ const txnData = {
+            pop_city: undefined,
+            pop_state_province: "",
+            pop_zip: undefined,
+            potential_total_value_of_award: "$227,269,217",
+            recipient_business_type: "Unknown Business Type",
+            recipient_city: "San Francisco",
+            recipient_country: "UNITED STATES",
+            recipient_duns: "177667227",
+            recipient_name: "MCKESSON CORPORATION",
+            recipient_parent_duns: "177667227",
+            recipient_state_province: "CA",
+            recipient_street: "ONE POST ST",
+            recipient_zip_postal: "94104",
+            total_obligation: "$227,269,217",
+            type: "Delivery Order",
+            type_description: "Delivery Order",
+            type_of_contract_pricing: "J",
+            type_of_contract_pricing_description: "Firm Fixed Price"
+        };
+
+        // Mock the call and validate it matches the expected award summary
+        const mockReduxAward = jest.fn((args) => {
+            expect(args).toEqual(expectedAwardData);
+        });
         // Set up container with mocked award action
         const awardContainer = shallow(
             <AwardContainer
                 params={parameters}
                 award={award}
-                setSelectedAward={AwardActions.setSelectedAward} />);
+                setSelectedAward={mockReduxAward} />);
 
         const parseAwardSpy = sinon.spy(awardContainer.instance(),
             'parseAward');
 
-        // Parse award
-        awardContainer.instance().parseAward(fullData);
+        // Run function on instance of container
+        awardContainer.instance().parseAward(awardData);
 
         // checking that it ran
         expect(parseAwardSpy.callCount).toEqual(1);
+        expect(mockReduxAward).toHaveBeenCalledTimes(1);
 
         // reset the spies
         parseAwardSpy.reset();
     });
 
     // parse transactions
     it('should parse the returned transactions and send to redux store', () => {
+        const expectedTxnData = [{
+            _jsid: "contract-transaction-3",
+            action_date: "10/1/2014",
+            action_type: null,
+            description: "EXPRESS REPORT PHARMACY PRIME VENDOR VA760PPVFY2015OCT",
+            federal_action_obligation: "$227,269,217",
+            id: 34779,
+            modification_number: "0",
+            type: null,
+            type_description: null
+        }];
+        // Mock the call and validate it matches the expected txn summary
+        const mockReduxAwardTxn = jest.fn((args) => {
+            expect(args).toEqual(expectedTxnData);
+        });
+
+        const mockAppendAwardTxn = jest.fn((args) => {
+            expect(args).toEqual(expectedTxnData);
+        });
+
+        const mockSetTxnMeta = jest.fn((args) => {
+            expect(args).toEqual(award.transactionMeta);
+        });
+
+        const mockUpdateTxnRenderHash = jest.fn((args) => {
+            expect(args).toEqual(undefined);
```

Comment:
Since these two functions don't take arguments, you can do an empty mock for them (`jest.fn();`) and then just check they they were called:
```
const mockUpdateTxnRenderHash = jest.fn();

expect(mockUpdateTxnRenderHash).toHaveBeenCalled();
```